docs/standard/assembly/reference-assemblies.md

```diff
@@ -0,0 +1,74 @@
+
+Using a reference assembly enables developers to build programs that target a specific library version without having the full implementation assembly for that version. Suppose, you have only the latest version of some library on your machine, but you want to build a program that targets a machine with an earlier version of that library. If you compile directly against the implementation assembly, you might inadvertently use API members that aren't available in the earlier version, and you'll only find this mistake when testing the program on the target machine. If you compile against the reference assembly for the earlier version, you'll immediately get a compile-time error.
+
+Additionally, a reference assembly can represent a contract, that is, a set of APIs that doesn't correspond to the concrete implementation assembly. Such reference assembly, called the *contract assembly*, can be used to target multiple platforms that support the same set of APIs. For example, .NET Standard provides the contract assembly, *netstandard.dll*, that represents the set of common APIs shared between different .NET platforms. The implementations of these APIs are contained in different assemblies on different platforms, such as *mscorlib.dll* on .NET Framework or *System.Private.CoreLib.dll* on .NET Core. A library that targets .NET Standard can run on all platforms that support .NET Standard. 
+
+## Using reference assemblies
+
+To use certain APIs from your project, you must add references to their assemblies. You can add references either to implementation assemblies directly or to reference assemblies. We strongly recommend that you use reference assemblies whenever they are available, because doing so ensures that you are using only API members that are supported in the target version and are meant to be used by API designers (in other words, not taking a dependency on implementation details).
+
+Reference assemblies for the .NET Framework libraries are distributed with targeting packs. You can obtain them by downloading a standalone installer or by selecting a component in Visual Studio installer. For more information, see [Install the .NET Framework for developers](../../framework/install/guide-for-developers.md). Reference assemblies for .NET Core libraries are managed as NuGet packages. For example, the [Microsoft.NETCore.App](https://www.nuget.org/packages/Microsoft.NETCore.App) package contains reference assemblies that represent common .NET Core APIs. To import these packages, you must first install the SDK for the target platform version you plan to use. For more information, see [Packages, metapackages and frameworks](../../core/packages.md) in the .NET Core Guide.
```

Comment:
@ericstj @livarcocc Is this description for .NET Core still accurate?

Reply:
There is at least one missing feature here. The SDK will auto-download the reference assemblies during restore assuming a specific property is set. @livarcocc or @nguerrera or @dsplaisted will know the property.

Reply:
For .NET Core, the SDK will download the targeting packs automatically if they aren't already available locally.  There is enabled by default, though you can set the `EnableTargetingPackDownload` property to false to disable.

Reply:
Also, for .NET Core 3.0, we don't use the Microsoft.NETCore.App package anymore.  It contained the reference assemblies but had references which would bring in the implementation.  In .NET Core 3.0, the targeting pack package name is Microsoft.NETCore.App.Ref.